about/policy/sgid/index.md

```diff
@@ -1,116 +1,171 @@
+- Datasets should have a reasonable **update schedule** included in the metadata.
+  - We may add datasets that result from a one-time project and won't be updated in the future, but this should be clearly stated in the metadata.
+- Datasets must not contain any information considered `private`, `protected`, or `controlled` under the Utah [Government Records Access and Management Act (GRAMA)](https://archives.utah.gov/opengovernment/) or any other personally identifiable information (PII).
 
-We will add a corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1). This will contain at a minimum:
+### SGID Index Qualifications
+{: .text-left}
 
-- The authoritative access from location
-- The table name
-- A description
-- The data steward
-- The steward contact name and email
-- A link to the data page on the AGRC website
-- A link to the SGID Open Data page
+These criteria apply to all datasets added to the SGID Index and not anywhere else in the SGID (for example, download links hosted on an entity's own server).
 
-We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding category folder and group and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`.
+- Datasets should be **publicly available** without requiring a login.
+  - We may add references to datasets that are restricted due to safety or security concerns as a way to raise awareness that the datasets exist but are restricted.
+- Dataset links should be **stable** and maintained for the forseeable future. We do not want to be constantly updating the Index due to dead links.
 
-The [Auditor](https://github.com/agrc/auditor) project will regularly synchronize the ArcGIS Online item's information (tags, thumbnail, title, metadata, etc.) with information from the SGID geodatabase to ensure consistency across all our platforms.
+### Open Data Qualifications
+{: .text-left}
 
-If the dataset is shared with our [Open Data site](https://opendata.gis.utah.gov/), we will enable the multiple formats download option on the ArcGIS Online item page to allow users to download File Geodatabases and other non-default download options.
+- Datasets must be **publicly available** without an ArcGIS login.
+- Datasets should have a **stable schema**. Clients might be using the services directly, and deleting or renaming fields can break pop-ups, definition queries, and other schema-dependent actions.
+  - Adding new fields is generally ok.
+- All datasets from a single entity must be shared from the same **ArcGIS Online user** OR have a custom and consistent organization name specified in the ArcGIS Online item metadata as described in the "Source" note in the [Open Data sharing process]({% link sharing/process/index.md %}#open-data).
```

Comment:
```suggestion
- All datasets from a single entity should be shared from the same **ArcGIS Online user** OR have a custom and consistent organization name specified in the ArcGIS Online item metadata as described in the "Source" note in the [Open Data sharing process]({% link sharing/process/index.md %}#open-data).
```
do we want a hard restriction on this?

Reply:
I vote yes because it helps keep the Open Data sources list intact, which has a couple benefits. 

- It presents a more authoritative view to the end user (which sounds more authoritative, a dataset shared by `joe.bob` or `Utah Department of Foo`?) 
- It also allows us to create URLs that sort by source, like https://opendata.gis.utah.gov/search?source=utah%20geological%20survey. This could be used to create a page of agency icons similar to the current page of category icons.